install.sh

```diff
@@ -1,37 +1,68 @@
 #!/usr/bin/env bash -e
-# Opengrep installation script 
+# Opengrep installation script
```

Comment:
I know this was like that, but it seems more portable to have:

```bash
#!/usr/bin/env bash
set -e
```

and maybe with: `set -euo pipefail` to be safer?